src/application.rs

```diff
@@ -41,11 +41,21 @@ pub trait ApplicationHandler {
     /// [`pageshow`]: https://developer.mozilla.org/en-US/docs/Web/API/Window/pageshow_event
     /// [`bfcache`]: https://web.dev/bfcache/
     ///
+    /// ### Android
+    ///
+    /// On Android, the [`resumed()`] method is called when the `Activity` is (again, if after a
+    /// prior [`suspended()`]) being displayed to the user.  This is a good place to begin drawing
+    /// visual elements, running animations, etc.  It is driven by Android's [`onStart()`] method.
```

Comment:
```suggestion
    /// visual elements, running animations, etc. It is driven by Android's [`onStart()`] method.
```

Reply:
Fair enough, separating individual sentences with a double-space is a habit that stuck with me since Linux kernel development days. A regex for `\.  ` shows that this only ever happened in two other places, so I've removed all these.